test/sequential/maintenance-pallet.api.rolldown.test.ts

```diff
@@ -240,11 +241,15 @@ describe.each(["mm", "upgradabilityMm"])(
         "%s operations are allowed in mm",
         async (testName) => {
           testLog.getLog().info("DEBUG::TestName - " + testName);
-          let signer: User;
+          let signer: User = getSudoUser();
           const [extrinsic, testsSigner] = tests[testName];
-          if (mmMode === "mm") {
+          if (mmMode === "mm" && testName === "sequencerTearDown") {
```

Comment:
sequencerTearDown for  "upgradabilityMm" doesn't pass and I don't understand why. Maybe we need to change signer from user to testSigner?

Reply:
I'll think on something else,